tools/getrealdeps.sh

```diff
@@ -61,13 +61,19 @@ if ! awk -W version &> /dev/null; then
   crew install mawk
 fi
 
+: "${CREW_PREFIX:=$(crew const | $GREP CREW_PREFIX | awk -F = '{print $2}')}"
```

Comment:
Avoid grep?
```
crew const CREW_PREFIX | cut -d= -f2
```

Reply:
Ah! That let me move that up in the file too...